### PR TITLE
📝 Rework local tethered development

### LINKED ISSUE #2909: 📝 Update tethered local dev
### Where on docs.platform.sh should be changed?

https://docs.platform.sh/development/local/tethered.html

### What exactly should be updated?

Take into account updates in Django local tethered dev guide. Possibly using shortcodes, but possibly not if too Python-specific.

Update for:

- Content accuracy
- Includes what it should
- Doesn't include what doesn't fit
- Fits template
- Fits style guide

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by CollierCZ**
## Why

Closes #2909

## What's changed

Updated shortcodes in the guide to be more universal and used them also in the tethered dev doc.

### CODE REVIEW

File: docs/layouts/shortcodes/guides/local-requirements.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 ## Before you begin
 
-You need:
+You need: {{ .Get "framework" }}
 
-- A local copy of the project repository on your computer.
+-   A local copy of the repository for a {{ with .Get "framework" }}[{{ . }}](../deploy/_index.md) {{ end }}project running on Platform.sh.
```
Comment (AnouckColson) [inline docs/layouts/shortcodes/guides/local-requirements.md:5]: 📋 

```suggestion
-   A local copy of the repository for a {{ with .Get "framework" }}[{{ . }}](../deploy/_index.md) {{ end }} project running on Platform.sh.
```

File: docs/layouts/shortcodes/guides/local-requirements.md [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
   
-  You can get one by running the following command: <code>platform get {{ `{{< variable "PROJECT_ID" >}}` | .Page.RenderString }}</code>.
-  Or clone an integrated source repository
-  and set the remote branch by running the following command: <code>platform project:set-remote {{ `{{< variable "PROJECT_ID" >}}` | .Page.RenderString }}</code>.
-- The [Platform.sh CLI](/administration/cli/_index.md).
\ No newline at end of file
+    You can get one by running the following command: <code>platform get {{ `{{< variable "PROJECT_ID" >}}` | .Page.RenderString }}</code>.
```
Comment (AnouckColson) [inline docs/layouts/shortcodes/guides/local-requirements.md:7]: 🛠️ 

```suggestion
    To get one, run the following command: <code>platform get {{ `{{< variable "PROJECT_ID" >}}` | .Page.RenderString }}</code>.
```

File: docs/layouts/shortcodes/local-dev/next-steps-end.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+1.  Commit and push the revisions by running the following command:
```
Comment (AnouckColson) [inline docs/layouts/shortcodes/local-dev/next-steps-end.md:1]: 🛠️ 

```suggestion
1.  To commit and push the revisions, run the following command:
```

File: docs/layouts/shortcodes/local-dev/next-steps-end.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+    ```bash
+    git add . && git commit -m "Add local configuration" && git push platform local-config
+    ```
+
+1.  Merge the change into production.
+
+Once the script is merged into production,
+any user can then set up their local environment by running the following commands:
```
Comment (AnouckColson) [inline docs/layouts/shortcodes/local-dev/next-steps-end.md:10]: ⛏️  Seems redundant with "once".

```suggestion
any user can set up their local environment by running the following commands:
```

File: docs/layouts/shortcodes/local-dev/next-steps-start.md [hunk 5]
```diff
@@ -15,10 +15,8 @@ You can merge this change into production.
 
 1.  Create a new environment called `local-config`.
 
-2.  Create an executable script to set up a local environment for a new Platform.sh environment. 
+1.  Create an executable script to set up a local environment for a new Platform.sh environment.
```
Comment (AnouckColson) [inline docs/layouts/shortcodes/local-dev/next-steps-start.md:18]:
```suggestion
1.  To set up a local environment for a new Platform.sh environment, create an executable script.
```

File: docs/layouts/shortcodes/tethered-dev/steps-start.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
+1.  Create a new environment based on production.
+
+    ```bash
+    platform branch new-feature {{ `{{< variable "PRODUCTION_ENVIRONMENT_NAME" >}}` | .Page.RenderString }}
+    ```
+
+    If you're using a [source integration](/integrations/source.html),
+    open a merge/pull request.
+
+2.  Open an SSH tunnel to the new environment's services by running the following command:
```
Comment (AnouckColson) [inline docs/layouts/shortcodes/tethered-dev/steps-start.md:10]:
```suggestion
2.  To open an SSH tunnel to the new environment's services, run the following command:
```

File: docs/src/development/local/tethered.md [hunk 7]
```diff
@@ -4,133 +4,68 @@ sidebarTitle: Tethered
...
-* For other languages it's recommended that you install your own copy of Nginx or Apache.
-* A virtual machine or Docker image is also a viable option.
+{{% tethered-dev/steps-end %}}
 
-## SSH tunneling
+## Connect to services directly
 
-Now that the code is running, it needs to connect it to its services. For that, open an SSH tunnel to the current project.
+With open tunnels to all your services, you can also connect to the running services directly.
+Get information on all running services by running the following command:
```
Comment (AnouckColson) [inline docs/src/development/local/tethered.md:34]: 🛠️ 
```suggestion
To get information on all running services, run the following command:
```

File: docs/src/guides/django/local/ddev.md [hunk 8]
```diff
@@ -194,7 +197,9 @@ hooks:
     ddev stop
     ```
 
-{{% guides/django/local-next-steps-start name="DDEV" %}}
+{{% local-dev/next-steps-start name="DDEV" %}}
+
+    Fill it with the following example, depending on your package manger:
```
Comment (AnouckColson) [inline docs/src/guides/django/local/ddev.md:202]: 📋 
```suggestion
    Fill it with the following example, depending on your package manager:
```

File: docs/src/guides/django/local/tethered.md [hunk 9]
```diff
@@ -25,36 +25,16 @@ The settings are used to mock the conditions of the environment locally.
...
-    mock two additional variables present in active environments:
+1.  To ensure your `settings.py` file acts as if in a Platform.sh environment,
+    mock two variables present in active environments:
 
     ```bash
     export PLATFORM_APPLICATION_NAME=django && export PLATFORM_ENVIRONMENT=new-feature
     ```
 
-4.  Install dependencies by running the command for your package manager:
+1.  Install dependencies by running the command for your package manager:
```
Comment (AnouckColson) [inline docs/src/guides/django/local/tethered.md:37]:
```suggestion
1.  To install dependencies, run the command for your package manager:
```

File: docs/src/guides/django/local/tethered.md [hunk 10]
```diff
@@ -132,13 +112,11 @@ poetry run python manage.py runserver
...
-7.  When you've finished your work, close the tunnels to your services by running the following command:
+{{% tethered-dev/steps-end %}}
 
-    ```bash
-    platform tunnel:close --all -y
-    ```
+{{% local-dev/next-steps-start %}}
 
-{{% guides/django/local-next-steps-start %}}
+    Fill it with the following example, depending on your package manger:
```
Comment (AnouckColson) [inline docs/src/guides/django/local/tethered.md:119]: 📋 
```suggestion
    Fill it with the following example, depending on your package manager:
```

**Review by AnouckColson — CHANGES_REQUESTED** [review]
A few minor changes here and there.

**Comment by CollierCZ** [comment]
I was clearly trying to go fast with this one. :see_no_evil: Thanks for the improvements!

**Pull request merged**